src/politech_backend/settings.py

```diff
@@ -241,6 +241,16 @@
     'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',)
 }
 
+SWAGGER_SETTINGS = {
```

Comment:
Does this have to be the authentication settings that Django runs?

Or could we f.x. use `oauth2` for Swagger only?